.github/workflows/a11y_tests.yaml

```diff
@@ -1,15 +1,17 @@
 name: Accessibility (a11y) Tests
 
 on:
-    pull_request:
+    workflow_dispatch:
+    # Disabled, as they are currently broken. See #3531
+    # pull_request:
 
 jobs:
     a11y_tests:
         strategy:
             fail-fast: false
             matrix:
-                php-version: ['7.2']
-                node-version: ['12.5']
+                php-version: ['8.2']
```

Comment:
The minimum PHP version must be changed in `composer.json` too?

Reply:
That is something for future work, as you can see this job has been disabled for now as it is completely broken anyways.